src/components/DatePicker/DatePickInput/DatePickInput.tsx

```diff
@@ -27,50 +34,97 @@ const DatePickInput = React.forwardRef<HTMLInputElement, Props & InputProps>(
   (
     {
       handleFocus,
+      buttonType,
+      styleType,
       handleClear,
       isRangePicker,
+      filterType,
       inputProps,
       selectedDay,
       dateFormatOverride = 'MM/DD/YYYY',
+      open,
     },
     ref
   ) => {
+    const theme = useTheme();
     const formatDate = (date: Dayjs | undefined) => {
       return date ? dayjs(date).format(getLocaleFormat(dateFormatOverride)) : '';
     };
 
     const getDateFormatted = React.useCallback(formatDate, []);
 
-    return (
-      <div css={rangeInputsWrapper()}>
-        {isRangePicker ? (
-          <TextField
-            ref={ref}
-            {...inputProps}
-            onFocus={handleFocus}
-            onKeyDown={handleClear}
-            onChange={ON_CHANGE_MOCK}
-            placeholder="Date (start) - Date (end)"
-            value={
+    const renderBase = React.useMemo(() => {
```

Comment:
using a useMemo here with all the properties taken from the parent component will get us nothing in return. We can skip that computation